src/main/java/org/jenkinsci/plugins/docker/commons/tools/DockerTool.java

```diff
@@ -114,7 +115,8 @@ public DockerTool forNode(Node node, TaskListener log) throws IOException, Inter
         }
     }
 
-    @Extension public static class DescriptorImpl extends ToolDescriptor<DockerTool> {
+    @Extension @Symbol("dockerTool")
```

Comment:
It was already the symbol used, so I'm not sure adding it is confusing?

Reply:
Just in case the algorithm for the default name changes in the future

Reply:
https://issues.jenkins.io/browse/JENKINS-37390